### PR TITLE
Initial upload

### PR DESCRIPTION
**Pull request by alevsh**

### CODE REVIEW

**Review by AnHolub — COMMENTED** [review]
reviewed

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+This page contains release notes information for Aspose.PDF for .NET 21.9.
+
+{{% /alert %}} 
+
+## Improvements and Changes
+
+|**Key**|**Summary**|**Category**|
+| :- | :- | :- |
+|PDFNET-48976 Support extended customization of Digital Signatures Appearance|Feature|
+|PDFNET-50158 Form fields cannot be read|Feature|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md:25]:
```suggestion
|PDFNET-50158 Collect data from the WidgetAnnotation|Feature|
```

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+|PDFNET-48659 API is taking long time while checking signatures|Enhancement|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md:26]:
```suggestion
|PDFNET-48659 Speed up while checking signatures|Enhancement|
```

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+|PDFNET-46513 API is taking long time to verify signatures from PDF document|Enhancement|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md:27]:
```suggestion
|PDFNET-46513 Speed up while verifying signatures from PDF document|Enhancement|
```

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+|PDFNET-48471 API is taking long time to convert PDF to images|Enhancement|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md:28]:
```suggestion
|PDFNET-48471 Speed up conversion PDF to images|Enhancement|
```

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+|PDFNET-49707 PDF to Docx: Recognize indents of paragraphs in EnchantedFlow mode|Enhancement|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md:29]:
```suggestion
|PDFNET-49707 PDF to DOCX: Recognize indents of paragraphs in EnchantedFlow mode|Enhancement|
```

**Review by andruhovski — CHANGES_REQUESTED** [review]

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md (hunk 1 above)
Comment (alevsh) [reply net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md]: Fixed

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md (hunk 2 above)
Comment (alevsh) [reply net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md]: Fixed

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md (hunk 3 above)
Comment (alevsh) [reply net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md]: Fixed

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md (hunk 4 above)
Comment (alevsh) [reply net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md]: Fixed

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md (hunk 5 above)
Comment (alevsh) [reply net/release-notes/release-notes-2021/aspose-pdf-for-net-21-9-release-notes/_index.md]: Fixed

**Pull request merged**